qiskit_ibm_runtime/ibm_backend.py

```diff
@@ -549,6 +581,272 @@ def __deepcopy__(self, _memo: dict = None) -> "IBMBackend":
         cpy._options = deepcopy(self._options, _memo)
         return cpy
 
+    def run(
+        self,
+        circuits: Union[QuantumCircuit, str, List[Union[QuantumCircuit, str]]],
+        dynamic: bool = None,
+        job_tags: Optional[List[str]] = None,
+        init_circuit: Optional[QuantumCircuit] = None,
+        init_num_resets: Optional[int] = None,
+        header: Optional[Dict] = None,
+        shots: Optional[Union[int, float]] = None,
+        memory: Optional[bool] = None,
+        meas_level: Optional[Union[int, MeasLevel]] = None,
+        meas_return: Optional[Union[str, MeasReturnType]] = None,
+        rep_delay: Optional[float] = None,
+        init_qubits: Optional[bool] = None,
+        use_measure_esp: Optional[bool] = None,
+        noise_model: Optional[Any] = None,
+        seed_simulator: Optional[int] = None,
+        **run_config: Dict,
```

Comment:
Do we want to remove `memory`, and `meas_return` as well as move `noise_model` and `seed_simulator` to the backend options as Blake mentioned [here](https://github.com/Qiskit/qiskit-ibm-provider/issues/640)?

And then have `memory=True` and `meas_return=MeasReturnType.SINGLE` as the default 
@jyu00

Reply:
Let's keep things the same for now. We can make the changes when we overhaul `backend.run`.